Source/WebCore/css/parser/CSSPropertyParserConsumer+Image.cpp

```diff
@@ -597,7 +603,7 @@ static RefPtr<CSSValue> consumePrefixedRadialGradient(CSSParserTokenRange& range
 // MARK: <linear-gradient()> | <repeating-linear-gradient()>
 // https://drafts.csswg.org/css-images-4/#linear-gradients
 
-static RefPtr<CSSValue> consumeLinearGradient(CSSParserTokenRange& range, const CSSParserContext& context, CSS::GradientRepeat repeating)
+template<CSSValueID Name> static RefPtr<CSSValue> consumeLinearGradient(CSSParserTokenRange& range, const CSSParserContext& context)
```

Comment:
How did you decide on template argument vs. function argument?

Reply:
I need the template argument to be a compile time constant for use in the return value, so I chose template.